lib/engine/part/edge.rb

```diff
@@ -7,6 +7,10 @@ module Part
     class Edge < Base
       attr_accessor :num
 
+      def id
+        @id ||= "#{hex.id}-#{@num}-#{@lanes[1]}"
```

Comment:
in the previous code didn't you have lanes[0] and lanes[1]

Reply:
let's do underscores here as well

Reply:
i see you did it in line 215 lanes[a or b]

Reply:
Lanes in Path is a two-dimensional array (array of arrays)

Reply:
The first dimension is "a" vs. "b", the second dimension is lane size and lane index